devel/cmake/Portfile

```diff
@@ -122,6 +122,13 @@ depends_lib-append  port:curl \
 # system libuv is the default as of 20160830 (g320f5)
 depends_lib-append  path:lib/pkgconfig/libuv.pc:libuv
 
+# configure is just a shell script that passes directly to bootstrap
+configure.cmd       ./bootstrap
+
+# bootstrap takes cmake-style args for the build of the cmake to be installed
+# the args are passed as usual for cmake, after "--", so we pass them as post_args
```

Comment:
These two comment lines look like a single sentence.  Maybe you can add a `;` at the end of the first line?